athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneMetadataHandlerTest.java

```diff
@@ -53,6 +53,7 @@
 
 import static org.junit.Assert.*;
 
+import static com.amazonaws.athena.connector.lambda.domain.predicate.Constraints.DEFAULT_NO_LIMIT;
```

Comment:
nit since this is test, this import not used

Reply:
I'll clean it up.

Reply:
done.